### ISSUE TITLE
VC 6.3 on EL7 misses rst2html

### ISSUE THREAD

**Issue by dmatetelki** [varnishcache/pkg-varnish-cache#129]
```
checking for rst2man-3.6... rst2man-3.4
configure: error: rst2html not found - (Weird, we found rst2man?!)
error: Bad exit status from /var/tmp/rpm-tmp.J0G8FS (%build)
    Bad exit status from /var/tmp/rpm-tmp.J0G8FS (%build)
```

**Comment by stone** [comment]
You need to depend on `python-docutils`

**Comment by dmatetelki** [comment]
`6.x` branch needs to have the `weelky`'s fixes.

**Comment by stone** [comment]
Just some clarifications.

> You need to depend on `python-docutils`
^ Which will give you `/usr/bin/rst2html` python2 version.

For python3 version you need to depend on `python34-docutils` 
and the script will be available at: `/usr/bin/rst2html-3.4`

**Comment by gquintard** [comment]
OBE
